Networking/Networking/Mapper/CustomerMapper.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+/// Mapper: Customer
+///
+struct CustomerMapper: Mapper {
+    /// We're injecting this field by copying it in after parsing responses, because `siteID` is not returned in any of the Customer endpoints.
+    ///
+    let siteID: Int64
+
+    /// (Attempts) to convert a dictionary into a `Customer` entity
+    ///
+    func map(response: Data) throws -> Customer {
+        let decoder = JSONDecoder()
+        decoder.dateDecodingStrategy = .formatted(DateFormatter.Defaults.dateTimeFormatter)
```

Comment:
This line sneaked through from my previous attempt to decode the response, but is not needed as we're not parsing the Dates into our model. Will remove it with the review 👍